doc/conf.py

```diff
@@ -1,13 +1,13 @@
 """Sphinx configuration file for an LSST stack package.
-
-This configuration only affects single-package Sphinx documenation builds.
+This configuration only affects single-package Sphinx documentation builds.
+For more information, see:
+https://developer.lsst.io/stack/building-single-package-docs.html
 """
 
-from documenteer.sphinxconfig.stackconf import build_package_configs
-import lsst.ap.association
+from documenteer.conf.pipelinespkg import *
```

Comment:
As a general comment (too late now given it's everywhere) it would seem less like magic and more PEP8 as:

```python
import documenteer.conf.pipelinespkg as p

p.project = "ap_association"
````

Reply:
Did you check this with a Sphinx run? I'm not sure this would be compatible with the namespacing that Sphinx expects for configurations.

Reply:
Aha. I'll take a look. I'm sure you are right -- namespace shenanigans seems right.

Reply:
Right. Falls over completely. I'll merge this.